deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -74,6 +74,9 @@ spec:
                   description: The name of the secret containing the BMC credentials
                     (requires keys "username" and "password").
                   type: string
+                disableCertificateVerification:
+                  description: Will skip certificate validation when true
```

Comment:
Was this CRD autogenerated from the code? I don't see this exact string anywhere, so I suspect not.
Also I guess we no longer test for this in CI :(